holidays/generate_ics.py

```
@@ -166,6 +177,27 @@ def validate_language(self) -> None:
                 "Use --list-languages to see supported values"
             )
 
+    def validate_output_template(self, placeholders: set[str]) -> None:
+        if not self.args.output_template:
+            return None
+
+        template = self.args.output_template
+        tokens = re.findall(r"[^{}]+|\{\{|\}\}|\{[a-z_]+\}", template)
+        if "".join(tokens) != template:
+            raise SystemExit("Invalid output template")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)